wire-compiler/src/test/java/com/squareup/wire/schema/RootTest.kt

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2018 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.schema
+
+import com.google.common.io.Closer
+import com.google.common.jimfs.Configuration
+import com.google.common.jimfs.Jimfs
+import okio.buffer
+import okio.sink
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.After
+import org.junit.Test
+import java.nio.file.FileSystem
+import java.nio.file.Files
+import java.util.zip.ZipEntry
+import java.util.zip.ZipOutputStream
+import kotlin.test.assertFailsWith
+import kotlin.text.Charsets.UTF_8
+
+class RootTest {
+  private val fs = Jimfs.newFileSystem(Configuration.unix())
+  private val closer = Closer.create()
```

Comment:
nit: explicit setup in @Before?

Reply:
I think it’s less code as-is? But I agree it’s asymmetric